sample-data/plugin/configuration.packages.json

```diff
@@ -2,7 +2,7 @@
   "packages": {
     "sk": {
       "PackageName": "EdFi.Ods.Extensions.Sk",
-      "PackageVersion": "1.0.184",
+      "PackageVersion": "1.0.247",
```

Comment:
@jamessmckay Change the extension package version in https://github.com/Ed-Fi-Alliance-OSS/Starter-Kit-SEA-Data-Collection/blob/SEASK-69/packer/scripts/sk.ps1 as well, otherwise packer build will break if started with clean build folder.